source/nodejs/adaptivecards-templating/docs/README.md

```diff
@@ -0,0 +1,13 @@
+[Adaptive Cards Templating Javascript SDK](README.md)
+
+# Adaptive Cards Templating Javascript SDK
+
+## Index
+
+### Classes
+
+* [Binding](classes/binding.md)
+* [EvaluationContext](classes/evaluationcontext.md)
+* [ExpressionParser](classes/expressionparser.md)
+* [GlobalSettings](classes/globalsettings.md)
+* [Template](classes/template.md)
```

Comment:
What does it take to consolidate this into a single API reference .md for easy reference to exported interfaces something like [https://docs.microsoft.com/en-us/dotnet/api/overview/azure/?view=azure-dotnet](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/?view=azure-dotnet)?

Reply:
I'll have to dig in to see what it'll take. It looked like it would be a bit more complicated than just generating from one level higher, though, so for now it seemed better to get docs working independently.